modules/ROOT/pages/configure-partner.adoc

```diff
@@ -128,7 +130,7 @@ Remember that there must always be at least one identifier per message flow befo
 
 After successfully deleting an identifier, you can create a new identifier with the same value as the one you deleted.
 
-You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identifiers.
+You can delete identifiers such as all X12 ISA, X12 GS, AS2, and DUNS identifiers. You cannot delete the last identifier.
```

Comment:
I don't understand the second sentence in light of the first, which says you can delete ALL identifiers.
And what is "the last identifier"? The last of each type in a type list, or the last listed for all types, or...?

Reply:
REworded.